### PR TITLE
add more to the documented function definition

### PR DESCRIPTION
**Pull request by zbtirrell**
Working with a customer, we needed these extra couple fields, but they weren't otherwise documented.

### CODE REVIEW

File: src/functions/advanced-functions/event.php [hunk 1]
```diff
@@ -33,8 +33,10 @@
 	 * - EventShowMap bool - Set to true to embed the map in the Event view.
 	 * - EventCost string - Default cost of the Event.
 	 * - EventURL string - Link to the Event Website or Third-Party page
+ 	 * - FeaturedImage string - URL or ID of a featured image
```
Comment (tr1b0t) [inline src/functions/advanced-functions/event.php]: :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: f3b103628f2247b71f57377a21e819ec</sub>

Comment (barryhughes) [inline src/functions/advanced-functions/event.php]: This is just an observation about the docblock in general—and not a comment on your change specifically—but, for cases like this, we should _probably_ be following this convention:

```php
/**
 * Create an event. ...
 *
 * @param array $args {
 *     @type string $EventStartDate Start date of event (required).
 *     @type string $EventEndDate End date of event (required).
 *     ...
 * }
 *
 * @otherTags
 */
```

I am pretty sure this jives nicely with WP Parser (thinking docs site). See: [make.wordpress.org "Inline Documentation Standards (parameters that are arrays)."](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#1-1-parameters-that-are-arrays)

Comment (lucatume) [reply src/functions/advanced-functions/event.php]: I confirm, proper doc block structure, while painful to type, has that structure.

Also: all doc-block lines should end in a full-stop, a question mark or an exclamation point.

File: src/functions/advanced-functions/event.php [hunk 2]
```diff
@@ -62,16 +103,19 @@ function tribe_create_event( $args ) {
...
+	 * @category Events
 	 *
-	 * @return int|bool ID of the event that was created. False if update failed.
 	 * @link     http://codex.wordpress.org/Function_Reference/wp_update_post
+	 *
 	 * @see      wp_update_post()
 	 * @see      tribe_create_event()
-	 * @category Events
+	 *
+	 * @param int|bool   $postId  ID of the event to be modified.
```
Comment (tr1b0t) [inline src/functions/advanced-functions/event.php:115]: :no_entry_sign: Expected 1 spaces after parameter type; 3 found

<sub>hash: c26d7fe54130f3ffbe43aa3078a731d3</sub>

File: src/functions/advanced-functions/event.php [hunk 3]
```diff
@@ -62,16 +103,19 @@ function tribe_create_event( $args ) {
...
+	 * @param array      $args    Args for updating the post. See {@link tribe_create_event()} for more info.
```
Comment (tr1b0t) [inline src/functions/advanced-functions/event.php:116]: :no_entry_sign: Expected 4 spaces after parameter type; 6 found

<sub>hash: 2b825b83791679d48f562959a7815e19</sub>

**Pull request merged**